packages/vue/src/examples/pages/home/Home.vue

```diff
@@ -6,6 +6,7 @@
         :key="index"
```

Comment:
There is no "popular categories" section and the sequence of sections is different on mobile. And there is typo in "match with it" sentence.